sonar-xml-plugin/src/test/java/org/sonar/plugins/xml/XmlSensorTest.java

```diff
@@ -140,6 +151,10 @@ public void should_not_execute_test_on_corrupted_file_and_should_not_raise_parsi
   }
 
   private void init(boolean activateParsingErrorCheck) throws Exception {
+    init(activateParsingErrorCheck, new File("src/test/resources"));
+  }
+
+  private void init(boolean activateParsingErrorCheck, File file) throws Exception {
```

Comment:
This `file` parameter is never used. Are you sure that's what you wanted to do?

I think you are supposed to call it `moduleBaseDir` (and I'm surprised it's working) and remove the next line. But I have the feeling it's not even required, as you are providing directly the file in the fileSystem.

Reply:
Actually, I investigated it a little bit more and you are right it is not required to provide the file as parameter in init function.